### ISSUE TITLE
Things to do

### ISSUE THREAD

**Issue by GoogleCodeExporter** [WhiteNeo/wiiflow#141]
```
1- Download titles.ini in others language, for example spanish.
2- Hability to disable 1 or 2 cheats from 6 cheats, now if you activate 
for example 6 cheats, yo can't disable 1 or 2 cheats from the list.
```

Original issue reported on code.google.com by `SiTW...@gmail.com` on 19 Mar 2010 at 12:22

**Comment by GoogleCodeExporter** [comment]
```
It download tittles.ini in spanish (Almost the original version)
That was configured in my translation
```

Original comment by `rvmn95` on 20 Mar 2010 at 5:26

**Comment by GoogleCodeExporter** [comment]
```
I think issue 1 is not an issue, since it already can download other languages. 
Also, the second one was an issue, but I think it's fixed in wiiflow 2.0. Can 
you please confirm this?
```

Original comment by `r-...@live.com` on 24 Jun 2010 at 6:04

**Comment by GoogleCodeExporter** [comment]
Original comment by `Miigotu` on 4 Jul 2010 at 6:17
- Changed state: **Fixed**
